heliopy/data/util.py

```diff
@@ -24,7 +25,8 @@
 
 def process(dirs, fnames, extension, local_base_dir, remote_base_url,
             download_func, processing_func, starttime, endtime,
-            try_download=True, units=None, processing_kwargs={}):
+            try_download=True, units=None,
+            units_func=None, processing_kwargs={}):
```

Comment:
The new `units_func` needs documentation adding. Probably best to copy the style of the `processing_func` entry below on line 70.

Reply:
Added something similar to the docstring for units_func.